README.md

```diff
@@ -8,6 +8,15 @@ It solves all known pitfalls that can disrupt your development.
 It also describes all guidelines, tips and tricks.
 If you want your pull requests (or patches) to be merged into master, please respect those guidelines.
 
+# How to build with JBake
+
+To build the drools-website (before and after your changes):
+
+```
+$ mvn clean generate-resources
+$ firefox target/website/index.html
```

Comment:
I'd really prefer we get away the idea of viewing the site as a file.

Please use the below instead:
```
mvn clean jbake:inline
firefox http://localhost:8820
```

Reply:
good catch, I documented the options I use in e8dcab6